webapp/views.py

```diff
@@ -506,6 +503,13 @@ def make_renewal(advantage, contract_info):
 
     renewal["renewable"] = False
 
+    if renewal["status"] == "done":
+        renewalModifiedDate = dateutil.parser.parse(renewal["lastModified"])
+        oneHourAgo = datetime.now(timezone.utc) - timedelta(hours=1)
+
+        if oneHourAgo < renewalModifiedDate:
```

Comment:
Perhaps better: `renewal["recently_renewed"] = oneHourAgo < renewalModifiedDate`, so that the key is present in the false case as well.